recipes/coprarna/meta.yaml

```diff
@@ -29,6 +29,7 @@ requirements:
     - python
     - suds-jurko ==0.6
     - coreutils ==8.25 #
+    - readline
```

Comment:
readline 6.2*
However, I'm not sure why this is needed. Coprana is not a C program, right?

Reply:
Hey, I got following error in testing with planemo:
/home/user/miniconda3/envs/__coprarna@2.1.0/lib/R/bin/exec/R: symbol lookup error: /home/user/miniconda3/envs/__coprarna@2.1.0/lib/R/bin/exec/../../lib/../../libreadline.so.6: undefined symbol: PC

Reply:
Can you update planemo or adjust your channel order? conda-forge should be infront defaults.